### ISSUE TITLE
Captcha - reCaptcha - Friendly Error Message & Retaining Form Field Values on Contact Form after unsuccessful reCAPTCHA V2 submission

### ISSUE THREAD

**Issue by NivF007** [joomla/joomla-cms#18450]
### Steps to reproduce the issue

When using a contact form and reCAPTCHA utilizing Joomla's native Contact component and Captcha - ReCaptcha plugin (set to version 2.0)

When the user fails to check the "I'm not a robot" checkbox, two undesirable things happen from a 'user experience' point-of-view.

 1. **Unfriendly/non-descript error message**: the returned error message states "Empty solution not-allowed."
 2. **Form Field Values reset:** all data entered by the user in the form fields is reset

### Expected result

Ideally, the Joomla site should provide a friendly descriptive error message, such as "reCAPTHA submission failed"  and retain the entered form field data.

### Actual result
Unfriendly, non-descript error message "**Error**: Empty solution not allowed."

Information in contact form fields cleared.

### System information (as much as possible)

LAMP on HostGator running PHP7 Joomla 3.8.1

**Comment by brianteeman** [comment]
Iirc that error message comes from Google not joomla

**Comment by Quy** [comment]
See PR #17743 for the form field values reset

**Comment by brianteeman** [comment]
Sorry I was wrong about the message

**Comment by ghost** [comment]
closed as having Pull Request #19051.

**Comment by roicarl** [comment]
I just installed Google reCaptcha V2 for my contact form. Am running under Joomla release 3.9.1. This is the error message I got if you don't click on the reCaptcha square-is this considered user-friendly (plus I lost my data)? Am I doing something wrong? 

Tried to post error message but keep getting errors here. The best way is to look at the form, fill it and then click send without clicking on the reCaptcha.

(https://www.northcountrysteakbuffet.com/contact-us.html)
